packages/components/pie-button/README.md

```diff
@@ -60,8 +61,17 @@ import { PieButton } from '@justeattakeaway/pie-button';
 // React
 // For React, you will need to import our React specific component build
 // Which wraps the web component using the @lit-labs/react package
+
+// Note: When using the React version of the component, please make sure
+// you also include React as a dependency in your project as well. See Peer Dependencies section.
+
 import { PieButton } from '@justeattakeaway/pie-button/dist/react';
 ```
+## Peer Dependencies
+
+Before using `@justeattakeaway/pie-button`, please make sure you have the following peer dependencies installed in your project:
+
+- `react` (for React integration only)
```

Comment:
There's some nice github markdown syntax you can use for notes to grab the reader's attention 😎 It looks like this:

> **Note**
> Before using `@justeattakeaway/pie-button`, please make sure you have the following peer dependencies installed in your project:
> - `react` (for integration with React apps only)

```suggestion
> **Note**
> Before using `@justeattakeaway/pie-button`, please make sure you have the following peer dependencies installed in your project:
> - `react` (for integration with React apps only)
```

Reply:
Nicee